### PR TITLE
Allow selecting multiple label values when creating silence.

### LINKED ISSUE #69: Allow a user to be able to add additional labels to a silence via alertmanager ui
It would be good to be able to specify labels that are not part of a currently firing alert so that we can reduce the number of silences that are needed.

For exampe I would like to do the following: 
`{
    "node": "foo", 
    "instance": "foo"
}`

on an alert where only instance is firing

### PR DESCRIPTION
**Pull request by prymitive**
This commit turns label checkboxes into multiselect dropdowns that are populated from all labels in current alert group. There is also a collapsible preview box that allows user to inspect raw silence JSON object we will send to the Alertmanager API. Fixes #69 

screen shot 2017-04-22 at 13 34 56

screen shot 2017-04-22 at 13 34 42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,17 @@ to alert data, therefore safe to be accessed by wider audience.
 Alertmanager's API isn't stable yet and can change between releases.
 unsee currently supports Alertmanager `0.4` and `0.5`.
 
+## Security
+
+unsee process doesn't send any API request to the Alertmanager that could modify
```
Comment (jamesog) [inline README.md]: I'd suggest a small amount or rewording here. Either "Unsee doesn't" or "The Unsee process doesn't". Same goes for the first bullet point. Then either "to Alertmanager" or "to the Alertmanager API".

File: README.md [hunk 2]
```diff
@@ -14,6 +14,17 @@ to alert data, therefore safe to be accessed by wider audience.
...
+alerts or silence state. But it does provide a web interface that allows a user
```
Comment (jamesog) [inline README.md]: Instead of starting a second sentence I think it should be a continuation, i.e. "or silence state, but it does"

Comment (prymitive) [inline README.md]: Is it better now?

**Comment by prymitive** [comment]
All commits require a review before merging to master, so I'm gonna keep bothering you, sorry.
I'm not much of a web developer so the practices on how to write proper JS code escapes me, I'm doing my best to make this code sane, but I can't really say that I'm happy about it yet.

I would rate my dislike for the quality of JS code in usee as **3**
pain-scale

File: README.md (hunk 2 above)
Comment (jamesog) [inline README.md]: 👍

**Pull request merged**